src/cephadm/cephadm

```diff
@@ -810,13 +810,7 @@ class CephIscsi(object):
         tcmu_container = get_container(self.ctx, self.fsid, self.daemon_type, self.daemon_id)
         tcmu_container.entrypoint = '/usr/bin/tcmu-runner'
         tcmu_container.cname = self.get_container_name(desc='tcmu')
-        # remove extra container args for tcmu container.
-        # extra args could cause issue with forking service type
```

Comment:
Why remove this comment?  It seems useful and still valid to me.

Reply:
put it back